### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by wiebken-n**
- adjusted bottom padding on all views + made impressum-footer non-sticky
- move-set toggle is now disabled of no set is active
- empty bed-space info now returns space in cm + styling has been adjusted
- implemented re-routing if logged-in/non-registered users try to go to view-paths for which they are not eligible (e.g. non-registered user tries to open plantbed-edit)

### CODE REVIEW

File: src/components/MoveSetInBed.vue
Comment (Colourpieces) [inline src/components/MoveSetInBed.vue:1]: I would do this the other way round. :disable=!isSetActive is easier to understand I think. Therefore it should be then also the following??
const isSetActive = computed(() => {
  return !(
    plantBedsStore.state.activeSets.find((bedItem) => bedItem.bedNumber === props.bedNumber) ===
    undefined
  )
})

File: src/views/BlogView.vue
Comment (Colourpieces) [inline src/views/BlogView.vue:1]: I think we won't need this class and the div here... we will see it when both develops are in the main

File: src/views/TodoView.vue
Comment (Colourpieces) [inline src/views/TodoView.vue:1]: I think we won't need this class and the div here... we will see it when both develops are in the main

**Review by Colourpieces — COMMENTED** [review]
See comment for MoveSetInBed.

File: src/components/MoveSetInBed.vue
Comment (wiebken-n) [reply src/components/MoveSetInBed.vue:1]: yep, I refactored it

File: src/views/BlogView.vue
Comment (wiebken-n) [reply src/views/BlogView.vue:1]: yep, I agree that we won't need it with the new component

File: src/views/TodoView.vue
Comment (wiebken-n) [reply src/views/TodoView.vue:1]: yep, I agree that we won't need it with the new component

**Pull request merged**